include/envoy/network/udp_packet_writer_config.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "envoy/config/typed_config.h"
+#include "envoy/network/udp_packet_writer_handler.h"
+
+#include "common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Network {
+
+class UdpPacketWriterConfigFactory : public Config::UntypedFactory {
```

Comment:
This class needs to inherits from Config::TypedFactory in order to do type based mapping.

Reply:
Using `TypedFactory` now!